client/src/Components/Pages/Static/TermsOfUse.js

```diff
@@ -490,23 +490,10 @@ export default class TermsOfUse extends Component {
                   international copyright and other intellectual property laws.
                 </Li>
                 <Li>
-                  All references in these Terms to ‘content’ includes any
-                  information or other material found on or via the earwig
-                  Platform, including without limitation text, databases,
-                  graphics, videos, software and all other features found on or
-                  via the earwig Platform.
+                  All references in these Terms to ‘content’ includes any information or other material found on or via the earwig Platform, including without limitation text, databases, graphics, audio files, videos, software and all other features found on or via the earwig Platform.
                 </Li>
                 <Li>
-                  We make the earwig Platform and our content available through
-                  the earwig Platform for your personal, non-commercial use
-                  only. You may view the earwig Platform’s pages and content
-                  online and may print a copy of these Terms and store such
-                  Terms in electronic form on your computer for your records,
-                  provided you keep intact any and all copyright and proprietary
-                  notices. You may not otherwise reproduce, modify, copy or
-                  distribute or use any of the content on the earwig Platform
-                  other than as expressly permitted under these Terms without
-                  our prior written consent.
+                  WWe make the earwig Platform and our content available through the earwig Platform for your personal, non-commercial use only. You may view the earwig Platform’s pages and content online, download and play on your personal device any audio files we make available for download from the Platform and print a copy of these Terms and store such Terms in electronic form on your computer for your records, provided you keep intact any and all copyright and proprietary notices. You may not otherwise reproduce, modify, copy or distribute or use any of the content on the earwig Platform other than as expressly permitted under these Terms without our prior written consent.
```

Comment:
```suggestion
                  We make the earwig Platform and our content available through the earwig Platform for your personal, non-commercial use only. You may view the earwig Platform’s pages and content online, download and play on your personal device any audio files we make available for download from the Platform and print a copy of these Terms and store such Terms in electronic form on your computer for your records, provided you keep intact any and all copyright and proprietary notices. You may not otherwise reproduce, modify, copy or distribute or use any of the content on the earwig Platform other than as expressly permitted under these Terms without our prior written consent.
```